### PR TITLE
Set up a new policy file for basic-test and capture-test

### PR DESCRIPTION
**Pull request by svjaffer**
zpr-a needs to be able to ping zpr-b on both of its addresses and vice-versa.

### CODE REVIEW

File: examples/milestone2/policies/basic-test.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      - [zpr.adapter.cn, eq, vs.zpr]
+    admin_attrs:
+      - [zpr.adapter.cn, eq, client.zpr.org]
+
+  datasources:
+    zpr.adapter:
+      api: validation/1
+      authority:
+        encoding: pem
+        cert_data: $import[../vs/ca-cert.pem]
```
Comment (mkolehmainen) [inline examples/milestone2/policies/basic-test.yaml:35]: This import here is resolved during compilation.  Is there actually a ca-cert.pem file in a directory above this named "vs"?  I would add the ca-cert.pem to the policies dir and then you don't need any "../vs/".  Or maybe I'm missing something?

Comment (mkolehmainen) [reply examples/milestone2/policies/basic-test.yaml:35]: Oh wait, you are putting this in examples milestone2?  Why not just add a "data" directory or something under `adapter/ph/test` ?  Anyway, I'm going to approve, but I think it maybe is a little odd to put this and the test related stuff in the milestone2 examples dir.

**Review by mkolehmainen — APPROVED** [review]
Psyched that this works! But have some reservations about locations of files.

**Comment by svjaffer** [comment]
See https://github.com/org-zpr/zpr-core/pull/585

**Pull request closed without merging**